zproject/settings.py

```diff
@@ -489,6 +489,7 @@ def get_secret(key):
     # Use fast password hashing for creating testing users when not
     # PRODUCTION.  Saves a bunch of time.
     PASSWORD_HASHERS = (
+                'django.contrib.auth.hashers.Argon2PasswordHasher',
```

Comment:
This config is only used in development!  This will just make the tests run slow.